addons/pos_sale/static/tests/tours/PosSaleTour.js

```diff
@@ -264,3 +265,16 @@ registry.category("web_tour.tours").add("PoSDownPaymentLinesPerTax", {
             }),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("PosShipLaterNoDefault", {
+    test: true,
+    steps: () =>
+        [
+            ProductScreen.confirmOpeningPopup(),
+            ProductScreen.clickQuotationButton(),
+            ProductScreen.selectFirstOrder(),
+            ProductScreen.clickPayButton(),
+            negateStep(PaymentScreen.shippingLaterHighlighted()),
+            PaymentScreen.isShown(),
```

Comment:
```suggestion
            PaymentScreen.isShown(),
            negateStep(PaymentScreen.shippingLaterHighlighted()),
```

You need to check if the payment screen is there, before you make a negated check.

Reply:
It's possible that the payment screen is not yet shown, the negated step will be satisfied.